tr/advanced/best-practice-security.md

```diff
@@ -150,53 +147,62 @@ app.use(session({
+
+## Bağımlılıklarınızın güvende olduğundan emin olun
 
-Using npm to manage your application's dependencies is powerful and convenient.  But the packages that you use may contain critical security vulnerabilities that could also affect your application.  The security of your app is only as strong as the "weakest link" in your dependencies.
+Uygulamanızın bağımlılıklarını yönetmek için npm kullanmak güçlü ve kullanışlıdır. Ancak kullandığınız paketler, uygulamanızı da etkileyebilecek kritik güvenlik açıkları içerebilir.  Uygulamanızın güvenliği, bağımlılıklarınızdaki "en zayıf halka" kadar güçlüdür.
 
-Since npm@6, npm automatically reviews every install request. Also you can use 'npm audit' to analyze your dependency tree.
+npm@6'dan beri npm otomatik olarak her yükleme isteğini inceler. Ayrıca 'npm audit' komutunu kullanarak bağımlılık ağacınızı analiz edebilirsiniz.
 
 ```sh
 $ npm audit
 ```
 
-If you want to stay more secure, consider [Snyk](https://snyk.io/).
+Daha fazla güvenli kalmak istiyorsanız, [Snyk](https://snyk.io/) aracını gözden geçirebilirsiniz.
 
-Snyk offers both a [command-line tool](https://www.npmjs.com/package/snyk) and a [Github integration](https://snyk.io/docs/github) that checks your application against [Snyk's open source vulnerability database](https://snyk.io/vuln/) for any known vulnerabilities in your dependencies. Install the CLI as follows:
+Bağımlılıklarınızdaki bilinen tüm güvenlik açıkları için [Synk'in açık kaynak güvenlik açığı veritabanı](https://snyk.io/vuln/)'na karşı uygulamanızı kontrol eden bir [komut satırı aracı](https://www.npmjs.com/package/snyk) ve de [Github integrasyonu](https://snyk.io/docs/github) sunar.
 
 ```sh
 $ npm install -g snyk
 $ cd your-app
 ```
 
-Use this command to test your application for vulnerabilities:
+Uygulamanızı güvenlik açıklarına karşı test etmek için bu komutu kullanın:
 
 ```sh
 $ snyk test
 ```
 
-Use this command to open a wizard that walks you through the process of applying updates or patches to fix the vulnerabilities that were found:
+Bulunan güvenlik açıklarını düzeltmek için güncelleme veya yama uygulama sürecinde size yol gösteren bir sihirbazı açmak için bu komutu kullanın:
 
 ```sh
 $ snyk wizard
 ```
 
-## Avoid other known vulnerabilities
+## Bilinen diğer güvenlik açıklarından kaçının
 
-Keep an eye out for [Node Security Project](https://npmjs.com/advisories) or [Snyk](https://snyk.io/vuln/) advisories that may affect Express or other modules that your app uses.  In general, these databases are excellent resources for knowledge and tools about Node security.
+Express'i veya uygulamanızın kullandığı diğer modülleri etkileyen [Snyk](https://snyk.io/vuln/) ve [Node Security Project](https://nodesecurity.io/advisories) tavsiyelerini takipte kalın. Genel olarak, bu veritabanları Node güvenliği hakkında bilgi ve araçlar için mükemmel kaynaklardır.
```

Comment:
The Node Security Project no longer exists. The copies of everything in the site should have these references already removed, which makes me suspect you accidentally translated an older copy of this page. You'll want to make sure it's up-to-date with the current English copy, if possible. Certainly remove this link, though, at a minimum.

Reply:
The English copy, in the file `en/advanced/best-practice-security.md`, still has a reference to The Node Security Project. I'm positive I've got the latest version before doing the work, and I checked the English copy it checks out with the Turkish copy I've translated.

Reply:
Oh, my bad. It looks like it goes to a different link. I can see in the diff you changed the link, though. Any reason why not keep the same link?

Reply:
As in, the English copy goes to https://npmjs.com/advisories but your changes here have it go to the non-existent https://nodesecurity.io/advisories

Reply:
Oh yes my bad, the link seems to be old. Fixed that one too.